### PR TITLE
feat: Added Condition for restricting pending txns

### PR DESCRIPTION
**Pull request by howdysuraj**
### Description
Added Change for toast message and dialog in my expenses list v2

Remove add more expenses from create new report as it is not working from day 0, refer this thread for discussion around this :  https://fylein.slack.com/archives/C017C400V37/p1709031202418129

## Clickup
https://app.clickup.com/t/86cuqqvvp

## UI Preview
https://www.loom.com/share/0139577e6f7d4b308d8b2bafd5473e9e?sid=f132021b-eb38-4b11-b48a-af41f976d7d0

Screenshot 2024-02-29 at 11 04 50 AM

### CODE REVIEW

File: src/app/core/services/platform/v1/shared/expenses.service.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ import { DateService } from '../../../date.service';
 export class ExpensesService {
   constructor(private dateService: DateService) {}
 
+  //TODO : ADD CONDITION BASED ON ORG SETTING
+  restrictPendingTransactionsEnabled(): boolean {
+    return false;
```
Comment (Julias0) [inline src/app/core/services/platform/v1/shared/expenses.service.ts:24]: Lets not merge these PRs till we get the org settings

Comment (howdysuraj) [reply src/app/core/services/platform/v1/shared/expenses.service.ts:24]: sure, thanks

File: src/app/core/services/platform/v1/shared/expenses.service.ts [hunk 2]
```diff
@@ -27,6 +32,15 @@ export class ExpensesService {
     return typeof expense.policy_amount === 'number' && expense.policy_amount < 0.0001;
   }
 
+  isExpenseInPendingState(expense: Expense): boolean {
```
Comment (Julias0) [inline src/app/core/services/platform/v1/shared/expenses.service.ts:35]: doesExpenseHavePendingCardTransaction

**Review by Julias0 — APPROVED** [review]

File: src/app/shared/components/create-new-report/create-new-report.component.html [hunk 3]
```diff
@@ -61,13 +61,6 @@
       >
       </app-expense-card>
     </div>
-
-    <div class="create-new-report--add-more-container" (click)="closeEvent()">
```
Comment (Julias0) [inline src/app/shared/components/create-new-report/create-new-report.component.html:65]: This is expected

**Pull request merged**